projects/bp-gallery/src/components/views/picture/sidebar/picture-info/PictureEditDialog.tsx

```diff
@@ -29,6 +29,7 @@ const PictureEditDialog = memo(function PictureEditDialog({
   const editorRef = useRef<TuiImageEditor | null>(null);
   const apolloClient = useApolloClient();
   const { calledViaLink } = useContext(PictureViewContext);
+  const imgSrc = useMemo(() => asUploadPath(picture.media), [picture.media]);
```

Comment:
Is there a reason you use `useMemo` for the `asUploadPath` here and nowhere else? If not, I think we should either use `useMemo` everywhere or nowhere.

Reply:
+1